e2e/test/scenarios/admin-2/whitelabel.cy.spec.js

```diff
@@ -210,10 +207,15 @@ describeEE("formatting > whitelabel", () => {
             { force: true },
           );
           cy.findByText("logo.jpeg").should("be.visible");
+
+          cy.findByText(
```

Comment:
assert that if we replace the broken image, the error should go away. This wasn't possible previously because you'll need to select the custom option to upload the image which would already remove the error message.